### PR TITLE
nvidia_x11.persistenced: fix build on master

### PR DESCRIPTION
**Pull request by cpcloud**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/os-specific/linux/nvidia-x11/persistenced.nix [hunk 1]
```diff
@@ -27,6 +32,9 @@ stdenv.mkDerivation rec {
       $out/bin/nvidia-persistenced
   '';
 
+  NIX_CFLAGS_COMPILE = [ "-I${libtirpc.dev}/include/tirpc" ];
+  NIX_LDFLAGS = [ "-L${libtirpc.dev}/lib" "-ltirpc" ];
```
Comment (jonringer) [inline pkgs/os-specific/linux/nvidia-x11/persistenced.nix:36]: this happens by default
```suggestion
  NIX_LDFLAGS = [ "-ltirpc" ];
```

I guess the others are still needed.

**Review by jonringer — COMMENTED** [review]
Please follow [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md) and [manual#submitting-changes-making-patches ](https://nixos.org/nixpkgs/manual/#submitting-changes-making-patches) and squash the fix-up commits.

This can be done without `git rebase -i` by doing:
```
git reset HEAD~1                    # move fix-up commits into unstaged
git add -- pkgs/                    # move changes into staged
git commit --amend --no-edit        # add changes to previous commit
git push ... ... --force            # modify current PR branch
```

However, `git rebase -i` is a more powerful alternative, I created a small video demonstrating it's use [here](https://www.youtube.com/watch?v=OR8Q-4PLalc). A more indepth text tutorial can be found [here](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase)

**Comment by cpcloud** [comment]
Cool, I am familiar with rebase :) Will do.

**Comment by jonringer** [comment]
it's a pre-created reply, too much info is better than too little

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 102963 [1](https://github.com/Mic92/nixpkgs-review)

**Pull request merged**